src/components/Home.js

```diff
@@ -130,13 +131,18 @@ export default function Home(props) {
               target="_blank"
               href="https://greenstand.org/treetracker/web-map"
             >
+              {/* Box Component sx attribute overrides the 
+                style within Link Component so be careful */}
+
               <Box
-                sx={{
-                  color: 'white',
-                  '.MuiBox-root&:hover': {
-                    color: '#adadad',
-                  },
-                }}
+                sx={
```

Comment:
I think these are fine so no need to change them here. But you did help me find a problem!

Reply:
I will revert back changes in Home.js then 🙂.